permafrost/tests.py

```diff
@@ -258,6 +264,37 @@ def test_delete_role_deletes_group(self):
         with self.assertRaises(Group.DoesNotExist):
             group = Group.objects.get(name=group_name)
 
+    def test_get_all_perms_for_all_categories(self):
+        all_perm_names = [perm.name for perm in get_all_perms_for_all_categories()]
+
+        # grab all the names from CATEGORIES in settings
+        category_perm_names = []
+        for category, category_data in CATEGORIES.items():
+            for optional_perm_data in category_data['optional']:
+                category_perm_names.append(optional_perm_data['label'])
+
+            for optional_perm_data in category_data['required']:
+                category_perm_names.append(optional_perm_data['label'])
+
+        for perm in category_perm_names:
+            self.assertIn(perm, all_perm_names)
```

Comment:
instead of doing a loop lets just can we just sort these lists and do an assertEqual right?

Reply:
agreed